internal/tokensList/tokensList.go

```diff
@@ -43,14 +43,6 @@ func BuildTokenList(chainID uint64) {
 	logs.Info(`Reloading tokenLists...`)
 
 	supportedTokenMap := make(map[string]DefaultTokenListToken)
-	for _, token := range AggregatedTokenList.Tokens {
-		supportedTokenMap[token.Address] = token
```

Comment:
This would make the following codes for Wido and Portal to be skipped, which I believe now would remove the need for the `saveTokensListToJSON()` to be running in the loop. Is this correct?